workloads/kube-burner/run_poddensity_test_fromgit.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/bash -e
+
+set -e
+
+export WORKLOAD=pod-density
+export METRICS_PROFILE=${METRICS_PROFILE:-metrics.yaml}
+export JOB_ITERATIONS=${JOB_ITERATIONS:-1000}
```

Comment:
This should be:
```
export JOB_ITERATIONS=${PODS:-1000} 
```

right?

Can you add this value to env.sh and reference its default value in README.md?

Reply:
Fixed